Source/control.cpp

```diff
@@ -323,7 +323,7 @@ static void DrawSpell(const CelOutputBuffer &out)
 static void PrintSBookHotkey(const CelOutputBuffer &out, Point position, const std::string &text)
 {
 	// Align the hot key text with the top-right corner of the spell icon
-	position += { SPLICONLENGTH - (GetLineWidth(text.c_str()) + 5), 17 - SPLICONLENGTH };
+	position += Size { SPLICONLENGTH - (GetLineWidth(text.c_str()) + 5), 17 - SPLICONLENGTH };

```

Comment:
This should be a `Point`, not `Size`. A size with negative height doesn't make sense.

Reply:
@glebm A `Point` that represents deltas also doesn't make sense. Since the values here are clearly lengths, I went for `Size` for now until we have a more appropriate type.

Reply:
We already have to clean up Point usage, introducing Size in places it does not belong adds more patterns that we would have to search for, and it is not as easy to track down as Point + Point as we can simply remove the operator overload for Point + Point, but if we want to retain it in some cases for Point + Size then we would have to manually find each of them case by case.